src/network/bridge.rs

```diff
@@ -288,12 +305,14 @@ impl driver::NetworkDriver for Bridge<'_> {
             return Ok(());
         }
 
-        match self.teardown_firewall(complete_teardown, bridge_name) {
-            Ok(_) => {}
-            Err(err) => {
-                error_list.push(err);
-            }
-        };
+        if let BridgeMode::L3 = mode {
```

Comment:
Can we simplify this together with the self.info.network.internal condition into one and then remove the early return from internal

Reply:
good idea, done